src/util/builders/url/base.director.ts

```diff
@@ -14,24 +14,38 @@ export default class BaseURLDirector {
     return urlBuilder.setDomain(this.baseDomain).setPath(path).build(true)
   }
 
-  buildRestApiURL(path?: string) {
+  buildWikimediaApiURL(path?: string) {
     return urlBuilder
       .setDomain(this.baseDomain)
       .setPath(path ?? 'api/rest_v1')
       .build(true, '/')
   }
 
-  buildDesktopRestApiURL(path?: string) {
+  buildDesktopApiUrl(path?: string) {
     return urlBuilder
       .setDomain(this.baseDomain)
       .setPath(path ?? 'api/rest_v1/page/html')
       .build(true, '/')
   }
 
+  buildMobileApiUrl(path?: string) {
```

Comment:
This is `WikimediaMobile`, naming of the end-point should be respected allong all the functions.

Reply:
Updated